motion-1/Overview.bs

```diff
@@ -92,30 +92,41 @@ In addition to the property-specific values listed in their definitions, all pro
 
 ISSUE: Add more details and examples.
 
-<h3 id="motion-path-property">Motion path: The 'motion-path' property</h3>
+<h3 id="offset-path-property">Define a path: The 'offset-path' property</h3>
 
 <pre class='propdef'>
-Name: motion-path
-Value: <<url>> | [ <<basic-shape>> | <<path()>> ] || <<geometry-box>> | none
-Initial: none
+Name: offset-path
+Value: <<url>> | <<angle>> && contain? | [ <<basic-shape>> | <<path()>> ] || <<geometry-box>> | none
+Initial: 0deg for <<angle>>, otherwise none
 Applies to: All elements. In SVG, it applies to <a>container elements</a> excluding the <{defs}> element and all <a>graphics elements</a>
 Inherited: no
 Percentages: n/a
 Computed value: as specified
 Media: visual
-Animatable: no
+Animatable: as specified for <a href="https://www.w3.org/TR/css3-transitions/#animatable-types">angle</a>, otherwise no
```

Comment:
How could I do this?
Would I keep the definition about animatable as it is and add a note about the restriction in the definition of << angle >>?
Or change the definition as follows:

> Animatable: as specified for <a href="https://www.w3.org/TR/css3-transitions/#animatable-types">angle</a>, otherwise no (This means that it is animatable only when the start and end values are angle values.)